cms/cms_toolbars.py

```diff
@@ -343,7 +344,7 @@ def add_logout_button(self, parent):
             action=admin_reverse('logout'),
             active=True,
             on_success=on_success,
-            method='GET',
+            method='POST',
```

Comment:
Is Csrf handled somewhere else?

Reply:
CMS JS frontend adds it automatically, but I think Django does not evaluate it.